manifests/04_oauthserver_trust_role.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
location on this file is incorrect.  This is about access to something managed *by* the operator, not something managed *for* the operator.  This should be created by the operator, not by the CVO.  Please move to bindata and use the staticresourcecontroller to create and maintain this.